### PR TITLE
various doc edits, including a few TODO-comments related to plotting

### PR DESCRIPTION
**Pull request by bqpd**
@whoburg, ready for review.

### CODE REVIEW

File: docs/source/gettingstarted.rst [hunk 1]
```diff
@@ -62,99 +55,121 @@ All other inputs follow those of the ``Variable`` class.
...
 
+::
+
+    The x dimension is 0.577351209028 meter.
+
 Sensitivities and dual variables
 ================================
 
-When a GP is solved, the solver returns not just the optimal value for the problem’s variables (known as the "primal solution") but also, as a side effect of the solving process, the effect that scaling the less-than side of each constraint would have on the overall objective (called the "dual solution", "shadow prices", or "posynomial sensitivities").
+When a GP is solved, the solver returns not just the optimal value for the problem’s variables (known as the "primal solution") but also, as a side effect of the solving process, the effect that scaling the :math:`\leq 1` of each canonical constraint would have on the overall objective (called the "dual solution", "shadow prices", or "posynomial sensitivities").
```
Comment (whoburg) [inline docs/source/gettingstarted.rst]: not sure these are called shadow prices in a GP context - I've only heard that for LPs

File: docs/source/gettingstarted.rst [hunk 2]
```diff
@@ -62,99 +55,121 @@ All other inputs follow those of the ``Variable`` class.
...
+
+From the dual solution we can compute the sensitivities for every fixed variable in the problem, which is often incredibly useful for seeing which of your constraints are most crucial, thus prioritizing your confirmation and remodeling of your assumptions.
```
Comment (whoburg) [inline docs/source/gettingstarted.rst]: I think we could delete the last bit of the sentence above, after the comma.

File: docs/source/gettingstarted.rst (hunk 1 above)
Comment (whoburg) [inline docs/source/gettingstarted.rst]: text above is somewhat repetitive with `Using variable sensitivities`

File: docs/source/gp101.rst [hunk 3]
```diff
@@ -4,9 +4,11 @@ Geometric Programming 101
 What is a GP?
 =============
 
-A Geometric Program (GP) is a special type of constrained non-linear optimization problem.
+A Geometric Program (GP) is a type constrained non-linear optimization problem.
 
-A GP is made up of special types of functions called *monomials* and *posynomials*. In the context of GP, a monomial is defined as:
+A GP consists of an objective and set of inequalities of a particular form, which define the solution for a set of variables which have strictly positive values (that is, they can't be zero).
```
Comment (whoburg) [inline docs/source/gp101.rst]: rewording this because `which define the solution` doesn't seem quite precise.

File: docs/source/gpkit101.rst [hunk 4]
```diff
@@ -5,14 +5,13 @@ GPkit is a Python package for defining and manipulating
 geometric programming (GP) models,
 abstracting away the backend solver.
 
-The primary goal of GPkit is to make it easy to create share and explore Geometric Programming models.
-Being fast and mathematically correct tend to align well with this goal.
+The goal of GPkit is to make it easy to create share and explore Geometric Programming models, which tends to align well with being fast and mathematically correct.
```
Comment (whoburg) [inline docs/source/gpkit101.rst]: drop second half of this sentence, after the comma (or reword the part about "mathematically correct")?

**Comment by whoburg** [comment]
@bqpd this is great!

Added one commit and a few comments, ready for re-review. I'm ready to merge.

**Comment by bqpd** [comment]
retest this please

**Comment by whoburg** [comment]
@bqpd, ok for me to rebase this onto the current origin/master and force push?

**Comment by bqpd** [comment]
Yes!

**Comment by whoburg** [comment]
ok will do that now

**Comment by whoburg** [comment]
rebased, update any local copies of this branch

**Comment by whoburg** [comment]
@bqpd, I'm ready to merge -- unless you're planning to add anything to this?

**Comment by bqpd** [comment]
Nope, go ahead! You'd mentioned the sensitivities explanation needed some work, so I was presuming you were waiting on that.

**Pull request merged**